app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :comments, foreign_key: :author_id
+  has_many :posts, foreign_key: :author_id
+
```

Comment:
- [ ] Good job on including the associations for `comments` and `posts` but a user also has a relationship to the `likes`. Please kindly include that